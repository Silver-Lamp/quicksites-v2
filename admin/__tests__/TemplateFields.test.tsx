import { render } from '@testing-library/react';
import TemplateFields from '../components/templates/TemplateFields';

it('matches snapshot', () => {
  const { asFragment } = render(
    <TemplateFields
      industry="towing"
      layout="default"
      color_scheme="blue-yellow"
      commitMessage="Snapshot test"
      onChange={() => {}}
    />
  );
  expect(asFragment()).toMatchSnapshot();
});
