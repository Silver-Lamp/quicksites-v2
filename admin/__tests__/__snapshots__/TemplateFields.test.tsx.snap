// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="grid grid-cols-2 gap-4"
  >
    <div>
      <label
        class="text-sm font-medium leading-none peer-disabled:cursor-not-allowed peer-disabled:opacity-70"
      >
        Industry
      </label>
      <input
        class="flex h-9 w-full rounded-md border border-input bg-transparent px-3 py-1 text-base shadow-sm transition-colors file:border-0 file:bg-transparent file:text-sm file:font-medium file:text-foreground placeholder:text-muted-foreground focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring disabled:cursor-not-allowed disabled:opacity-50 md:text-sm"
        value="towing"
      />
    </div>
    <div>
      <label
        class="text-sm font-medium leading-none peer-disabled:cursor-not-allowed peer-disabled:opacity-70"
      >
        Layout
      </label>
      <input
        class="flex h-9 w-full rounded-md border border-input bg-transparent px-3 py-1 text-base shadow-sm transition-colors file:border-0 file:bg-transparent file:text-sm file:font-medium file:text-foreground placeholder:text-muted-foreground focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring disabled:cursor-not-allowed disabled:opacity-50 md:text-sm"
        value="default"
      />
    </div>
    <div>
      <label
        class="text-sm font-medium leading-none peer-disabled:cursor-not-allowed peer-disabled:opacity-70"
      >
        Color Scheme
      </label>
      <input
        class="flex h-9 w-full rounded-md border border-input bg-transparent px-3 py-1 text-base shadow-sm transition-colors file:border-0 file:bg-transparent file:text-sm file:font-medium file:text-foreground placeholder:text-muted-foreground focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring disabled:cursor-not-allowed disabled:opacity-50 md:text-sm"
        value="blue-yellow"
      />
    </div>
    <div>
      <label
        class="text-sm font-medium leading-none peer-disabled:cursor-not-allowed peer-disabled:opacity-70"
        for="commitMessage"
      >
        Commit Message
      </label>
      <input
        class="flex h-9 w-full rounded-md border border-input bg-transparent px-3 py-1 text-base shadow-sm transition-colors file:border-0 file:bg-transparent file:text-sm file:font-medium file:text-foreground placeholder:text-muted-foreground focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring disabled:cursor-not-allowed disabled:opacity-50 md:text-sm"
        id="commitMessage"
        placeholder="e.g. Updated CTA text on hero"
        type="text"
        value="Snapshot test"
      />
    </div>
  </div>
</DocumentFragment>
`;
